src/backend/core/models.py

```diff
@@ -1091,3 +1092,33 @@ def get_abilities(self, user):
             "patch": False,
             "put": False,
         }
+
+
+class AccountService(BaseModel):
+    """Account service model."""
+
+    name = models.CharField(_("name"), max_length=255)
+    api_key = models.CharField(
+        _("api key"),
+        max_length=255,
+        default=secrets.token_urlsafe(32),
+        editable=False,
```

Comment:
Why not?

Reply:
Because it's an automatically generated token. It makes sense to me. But whatever.